casper/src/test/scala/coop/rchain/casper/ValidateTest.scala

```diff
@@ -762,7 +757,8 @@ class ValidateTest
                    genesis.withBody(
                      genesis.body.get
                        .withDeploys(
-                         genesis.body.get.deploys.map(_.withLog(List(Event(EventInstance.Empty))))
+                         genesis.body.get.deploys
+                           .map(_.withDeployLog(List(Event(EventInstance.Empty))))
```

Comment:
a copy of this assertion for paymentLog is needed

Reply:
See my arguments above.